### DISCUSSION TITLE
Does gpt-engineer only develop python applications?

### DISCUSSION THREAD

**Discussion by Bubbalubagus** [AntonOsika/gpt-engineer#963] [Q&A]
Does gpt-engineer only develop python applications? It looks like it but would would be stopping this structure and app from developing all sorts of applications?

**Comment by ATheorell** [comment]
In theory, gpt-engineer can write any coding language. Python and javascript are the most tested ones. Try and add your desired language to the prompt. How well it works will depend on how well the LLM knows the specific language.
